taxi/forms.py

```diff
@@ -0,0 +1,42 @@
+from django import forms
+from django.core.exceptions import ValidationError
+from django.contrib.auth.forms import UserCreationForm
+
+from .models import Driver, Car
```

Comment:
generally, absolute imports are more reliable and should be preferred. From python docs:
> Absolute imports are recommended, as they are usually more readable and tend to be better behaved (or at least give better error messages) if the import system is incorrectly configured (such as when a directory inside a package ends up on sys.path):